### DISCUSSION TITLE
I deployed Airbyte through K*s and want to secure the webapp with user name and password authentication ?using GCP as a service

### DISCUSSION THREAD

**Discussion by bismayjena** [airbytehq/airbyte#33486] [Connector Questions] [answered]
I deployed Airbyte through Kubernetes and want to secure the webapp with user name and password authentication ?using GCP as a service

**Comment by pgrant87** [comment]
Hi @bismayjena If you have deployed via helm, you should be able to configure airbyte via the `values.yaml` file as per https://docs.airbyte.com/operator-guides/configuring-airbyte#kubernetes-deployments

The `access section` has the settings for username and password:
https://docs.airbyte.com/operator-guides/configuring-airbyte#access

**Reply by bismayjena** [reply]
I check there is no settings for user name and password from kubernetes ,yes from docker it is there

**Reply by bismayjena** [reply]
it is for the docker 

BASIC_AUTH_USERNAME=airbyte
BASIC_AUTH_PASSWORD=password
BASIC_AUTH_PROXY_TIMEOUT=600 

but how can we configure for the kubernetes through values.yaml

**Reply by marcosmarxm** [reply, marked as answer]
The airbyte-proxy doesn't work in the kubernetes deployment. To secure your app you need to manage the ingress access to the application yourself. There are some plans to release in the future better integration with keycloack and other access tooling.
